src/NHibernate.Test/Async/NHSpecificTest/NH1171/Fixture.cs

```diff
@@ -21,7 +21,12 @@ public class FixtureAsync : BugTestCase
 		protected override bool AppliesTo(Dialect.Dialect dialect)
 		{
 			// Firebird has issues with comments containing apostrophes
-			return !(dialect is FirebirdDialect);
+			return !(dialect is FirebirdDialect)
+#if !NETFX
```

Comment:
Let's remove the `#if`s and update managed .NET (not core) provider to latest version as well.